### ISSUE TITLE
Python3 compatibility for pipelines

### ISSUE THREAD

**Issue by TheChymera** [IBT-FMI/SAMRI#22]
I keep my data analysis server on Python 2.7, so I have not yet tested the pipelines on `3.*`. @damaggu do you have any Python `3.*` machine which you could test the pipelines on?

Certainly the relative imports need to be updated (“Implicit relative imports should never be used and have been removed in Python 3” - from [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports)), but there may be other issues as well.

**Comment by TheChymera** [comment]
d7f14b02a0c9f4dcea3fa245d141ace0cd9a296e might have been all of them

**Comment by TheChymera** [comment]
In a related matter, [nipype will release a new API, working only with python 3 at the end of the year](https://github.com/nipy/nipype/issues/1983#issuecomment-300302837). Ideally we would move our pipelines from the present API to the new one in between the submission for ISMRM 2018 and the actual conference.

**Comment by damaggu** [comment]
Yes, I'll set up a machine to test with python 3.6 !

**Comment by TheChymera** [comment]
How is this coming along? While I believe I have been coding Python3-compatibly, I am still doing all the heavy-lifting on Python2.7 systems. Have you executed all of your analysis for the ISMRM abstract on Python3?

**Comment by TheChymera** [comment]
The main deployments of SAMRI are now on python 3.6, in fact, with the pending deprtecation of python 2.* and the introduction of 3D plots in SAMRI via blender, which lacks python 2.* support:

```
chymera@zenhost ~ $ eix blender
[I] media-gfx/blender
     Available versions:  2.72b-r4 (~)2.79 2.79-r1 2.79b-r1 {+boost +bullet collada colorio cuda cycles +dds debug doc +elbeem ffmpeg fftw +game-engine headless jack jemalloc jpeg2k libav llvm man ndof nls openal opencl +openexr openimageio openmp +opennl opensubdiv openvdb osl player redcode sdl sndfile test tiff valgrind CPU_FLAGS_X86="sse sse2" PYTHON_SINGLE_TARGET="python3_5 python3_6" PYTHON_TARGETS="python3_4 python3_5 python3_6"}
     Installed versions:  2.79b-r1(17:27:38 18/02/19)(bullet cycles dds elbeem game-engine nls openexr openimageio openmp sdl tiff -collada -colorio -cuda -debug -doc -ffmpeg -fftw -headless -jack -jemalloc -jpeg2k -libav -llvm -man -ndof -openal -opencl -opensubdiv -openvdb -osl -player -sndfile -test -valgrind PYTHON_SINGLE_TARGET="python3_6 -python3_5" PYTHON_TARGETS="python3_6 -python3_5")
     Homepage:            https://www.blender.org
     Description:         3D Creation/Animation/Publishing System
```

It looks like we can in fact no longer guarantee that this will work on python 2.* we should start dropping support soon.
